website/aws.erb

```diff
@@ -342,7 +342,10 @@
                          <li>
                             <a href="/docs/providers/aws/d/network_interfaces.html">aws_network_interfaces</a>
                         </li>
-                        <li>
+                         <li<%= sidebar_current("docs-aws-datasource-organizations-organization") %>>
```

Comment:
Oh! We no longer need `sidebar_current` in `website/aws.erb` from these older pull requests. 👍 

```suggestion
                        <li>
```